src/tactic/ui/input/text_input_wdg.py

```diff
@@ -280,16 +280,17 @@ def fill_data(my):
                 sobject = Search.get_by_search_key(search_key)
 
             if sobject:
-
-                # look at the current sobject for the data
-                column = my.kwargs.get("column")
-                if not column:
-                    column = my.name
-
-                display = sobject.get_value(column, no_exception=True)
+            # look at the current sobject for the data
+                if not sobject.is_insert():
+                    column = my.kwargs.get("column")
+                    if not column:
+                        column = my.name
+                    
+                    display = sobject.get_value(column, no_exception=True)
+                display = ""
```

Comment:
Why is this always empty when you have a condition that is retrieving a value for display: display = sobject.get_value(column, no_exception=True). Even if you retrieve a value, it is always empty